rollup/internal/controller/sender/estimategas.go

```diff
@@ -76,7 +76,5 @@ func (s *Sender) estimateGasLimit(opts *bind.TransactOpts, contract *common.Addr
 		gasLimit = minGasLimit
 	}
 
-	gasLimit = gasLimit * 15 / 10 // 50% extra gas to avoid out of gas error
```

Comment:
you should not remove it. Or did you multiple 1.2 outside?

Reply:
yes. before pass in minGasLimit. so I should remove 1.2 outside?

Reply:
fixed in [552bf35](https://github.com/scroll-tech/scroll/pull/939/commits/552bf35436e1140edd1db70a91568963f047bc37)

Reply:
refactored this PR. I'll resolve this comment first.